.github/workflows/continuousIntegration.yml

```diff
@@ -0,0 +1,56 @@
+name: Continuous integration
+on:
+  create:
+  push:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: ['7.3']
```

Comment:
Please include PHP 7.4 as well, I hope to migrate to PHP 7.4 later this year (PHP 8 support is not present in MediaWiki yet).

Reply:
Technically this file just says edited because it was added in a merge of the master branch to this one, so I'll do that after merging this so I don't merge conflict things further.